src/main/kotlin/org/digma/intellij/plugin/ui/frameworks/SpringBootMicrometerConfigureDepsPanel.kt

```diff
@@ -0,0 +1,109 @@
+package org.digma.intellij.plugin.ui.frameworks
+
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import com.intellij.ui.components.ActionLink
+import com.intellij.util.Alarm
+import com.intellij.util.AlarmFactory
+import com.intellij.util.ui.JBUI
+import org.digma.intellij.plugin.idea.frameworks.SpringBootMicrometerConfigureDepsService
+import org.digma.intellij.plugin.ui.common.Laf
+import org.digma.intellij.plugin.ui.common.asHtml
+import org.digma.intellij.plugin.ui.common.buildLinkTextWithUnderLine
+import org.digma.intellij.plugin.ui.common.spanBold
+import org.digma.intellij.plugin.ui.panels.DigmaResettablePanel
+import javax.swing.BorderFactory
+import javax.swing.Box
+import javax.swing.BoxLayout
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+class SpringBootMicrometerConfigureDepsPanel(
+    val project: Project,
+) : DigmaResettablePanel(), Disposable {
+
+    init {
+        theService().affectedPanel = this
```

Comment:
"theService()"?

Reply:
this view has only ONE service it works with